### ISSUE TITLE
spicetify backup apply not working with newest version

### ISSUE THREAD

**Issue by operationairstrike** [spicetify/spicetify-cli#2966]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 1.2.35.663.gb699649e
- Spicetify version: 2.36.4
- Windows 11 newest
- Source: Spotify Website
```

### 📝 Description

It gets stuck on:
Preprocessing:
info Fetching remote CSS map for newer compatible tag version: master

when running spicetify backup apply

### 📸 Screenshots

![grafik](https://github.com/spicetify/spicetify-cli/assets/127621147/fae6b934-1021-47a0-9ce9-88f3b104e4a9)

**Comment by rxri** [comment]
Yes it does. If you’re stuck at this step you’re ratelimited by github. Wait a few minutes

**Comment by operationairstrike** [comment]
No it was first try bro

**Comment by ohitstom** [comment]
> I could access it over curl and firefox

this step is extremely slow if you are having issues wait up to ten minutes.

if that still doesn't work backup apply with no internet and it will revert to using the local CSS map

**Comment by operationairstrike** [comment]
It works on my personal computer, but not on my work laptop, that's weird I'm using Google DNS and I can access from curl and browser as said.
